packages/lavapack/src/runtime-cjs.js

```diff
@@ -1,8 +1,139 @@
 ;(function() {
```

Comment:
`runtime-cjs.js` is from now on a generated file (the original old one is now `runtime-cjs-template.js`)

Reply:
While this applies equally to the existing `runtime.js`/`runtime-template.js` and due to consistency is probably out of scope for this PR, mixing source and built files in the `src` directory is a bit confusing. Maybe `runtime.js` and `runtime-cjs.js` could be in a separate directory?

In either case, what do you think of having `builder-runtime.js` prepend a comment with a message like below to the generated file?
```
// DO NOT EDIT! THIS FILE IS GENERATED FROM ./runtime-cjs-template.js BY ./builder-runtime.js
```

Reply:
693d0a4

Reply:
And agreed, but also agreed that relocating those files should be done in a different PR